docker/ml/Dockerfile

```diff
@@ -1,14 +1,18 @@
 FROM demisto/python3-deb:3.8.6.14507
-
 COPY requirements.txt .
-RUN apt-get update && apt-get install -y --no-install-recommends python3-dev g++\
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+  gcc \
+  python3-dev gfortran wget git g++ pkg-config libhdf5-dev \
+&& ln -s /usr/include/locale.h /usr/include/xlocale.h \
 && pip install --no-cache-dir -r requirements.txt \
 && apt-get purge -y --auto-remove \
-  g++ \
-  python3-dev
+  gcc \
+  python3-dev gfortran wget git g++ pkg-config libhdf5-dev
```

Comment:
Why are you installing **gcc python3-dev gfortran wget git g++ pkg-config libhdf5-dev** twice ? 
I think you can install them and then remove them  no ? If not you can install them once after **apt-get purge -y --auto-remove**